### DISCUSSION TITLE
Jaguar Team Tap

### DISCUSSION THREAD

**Discussion by pkerga** [darthcloud/BlueRetro#200] [Q&A] [answered]
Hello
What does TeamTap jaguar support mean? Is it possible to associate in this way 2 controlers on the same jaguar with 1 ESP 32? Or is it only for TeamTap compatible games (2 games I believe by the way...) : Can I Play NBA Jam with 2 controlers and one ESP32 ? Thanks

**Comment by darthcloud** [comment, marked as answer]
It means it emulate a team tap and you can connect up to 4 controller to the same adapter.

It obviously only work with game that support team tap.

Also those games expect the team tap in 2nd port, so with only one adapter at best you have P1 using a wired controller and with the adapter on port 2 you can have an extra 4P (but games only use 2 or 3)

**Reply by pkerga** [reply]
and is it possible to connect several players to a PCE BlueRetro receiver?

**Reply by darthcloud** [reply]
See https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-System-Specific-User-Manual#pce--tg16

As described in main page readme:
BlueRetro is a multiplayer Bluetooth controllers adapter for various retro game consoles.

So multiplayer is supported for all systems.

**Reply by pkerga** [reply]
Thanks, I hadn't seen that interesting WIKI part. In a concrete way, I was able to play Mariokart with 4 bluetooth controllers on my N64 with 1 ESP32. Can I play with 5 Bluetooth controllers in Bomberman with 1 ESP32 on my PCE? (Sorry if my question is redundant, not sure of my English level 10 years old)

**Reply by darthcloud** [reply]
Yes 5 player, but PCE is currently broken so wait for next release.

**Reply by pkerga** [reply]
By the time I manage to install the wiring, you may have arrived at version 2 :-))
Anyway, this is my next project!
Is there an older version of BlueRetro in which the PCE works?
